PowerShellTools/PowerShellTools.vsct

```diff
@@ -5,8 +5,8 @@
   <Extern href="vsshlids.h"/>
 
   <KeyBindings>
-    <KeyBinding guid="guidPoshToolsCmdSet" id="cmdidExecuteAsScript" editor="guidVSStd97" key1="VK_F5" mod1="Alt Shift"/>
-    <KeyBinding guid="guidPoshToolsCmdSet" id="cmdidExecuteWithParametersAsScript" editor="guidVSStd97" key1="VK_F5" mod1="Control Alt Shift"/>
+    <KeyBinding guid="guidPoshToolsCmdSet" id="cmdidExecuteAsScript" editor="guidVSStd97" key1="0x45" key2="VK_F5" mod1="Shift"/>

```

Comment:
What's 0x45?  we can't use the letter itself?
https://msdn.microsoft.com/en-us/library/bb491775.aspx